include/message_filters/sync_policies/approximate_epsilon_time.h

```diff
@@ -0,0 +1,357 @@
+/*********************************************************************
+* Software License Agreement (BSD License)
+*
+*  Copyright (c) 2009, Willow Garage, Inc.
```

Comment:
Other maintainers can correct me otherwise, but I think our preference is to add "Open Source Robotics Foundation, Inc." as the copyright holder for new/significant contributions. I think that applies here.

Reply:
Sounds good, fixed in https://github.com/ros2/message_filters/pull/84/commits/bd38a7030d8195c180df01e2947c5069bc588d8b.
I'm still keeping BSD, as it's the license used in the rest of the package, but if I should update to Apache 2.0 let me know

Reply:
I think BSD is correct.